packages/core/lib/commands/debug/meta.js

```diff
@@ -59,7 +63,11 @@ module.exports = {
         option: "--compile-none",
         description:
           "Forces the debugger to use artifacts even if it detects a problem.  Dangerous; may cause errors."
-      }
+      },
+      {
+        option: "--url",
+        description: "Sets the server URL inline and ignore truffle config file"
```

Comment:
Again, I'd reword this description.  I'll suggest something like "Connects to a specified provider given via URL, ignoring networks in config"